doc/content/getting-started/migrating-from-v2/configure-mac-settings.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Fine-tuning MAC Settings for End Devices
+weight: 60
+---
+
+{{< cli-only >}}
+
+This section is relevant for end devices that need MAC settings fine-tuning to function properly with the Network Server of {{% tts %}}.
+
+MAC settings on {{% tts %}} are configurable per end device. They can be configured from the CLI.
+
+### Setting RX1 Delay
+
+The RX1 delay of end devices is set to 1 second by default. For some end devices, this may lead to traffic being dropped. In that case, it is recommended that the RX1 delay be increased to 5 seconds:
+
+```bash
+$ ttn-lw-cli end-devices update "app-id" "device-id" --mac-settings.rx1-delay RX_DELAY_5
+```
+
+> *NOTE*: For ABP end devices, you may have to reset the device for the changes to take effect.
```

Comment:
Again, what does this mean?

If the ABP device resets this state, and the RX1 delay mismatches with the NS MAC state, we don't have a downlink path at all. I would argue that this setting is not applicable to ABP devices at all, as the RX1 delay is part of the personalization. For ABP, you have to set this setting both in NS and end device.

Reply:
> If the ABP device resets this state, and the RX1 delay mismatches with the NS MAC state, we don't have a downlink path at all

Network Server will reset stored device MAC state and get the correct RX1 delay so we will, in fact, have a downlink path.
But indeed, in majority of cases `mac-settings.rx1-delay` MUST be correct at *creation* time of the device and updating this is quite tricky(possible though, by changing `mac-state.current-parameters.rx1-delay` *and* `mac-settings.rx1-delay`)